src/tutorial/runtime/introduction.rst

```diff
@@ -472,15 +474,13 @@ Files created by ``cylc install``
 
    Cylc generates files and directories when it installs a workflow:
 
-   ``log/``
-      ``log/install``
-         A record of the installation of this workflow.
+   ``log/install/``
+      A record of the installation of this workflow.
 
 
 .. ifslides::
 
-   * ``log/``
-      * ``log/install``
+   * ``log/install/``
 
```

Comment:
There are some more of these below, while you're at it. Would be nice to fix the inconsistency at ``flow-config/flow.cylc.processed``
```rst
   ``log``
      ``log/db``
         The database which Cylc uses to record the state of the workflow;
      ``log/job``
         The directory where the :term:`job log files <job log>` live;
      ``log/workflow``
         The directory where the :term:`workflow log files <workflow log>` live.
         These files are written by Cylc as the workflow is run and are useful for
         debugging purposes in the event of error.
      ``flow-config/flow.cylc.processed``
         A copy of the :cylc:conf:`flow.cylc` file made after any `Jinja2`_ has been
         processed - we will cover this in the
         :ref:`tutorial-cylc-consolidating-configuration` section.
```